docs/03-tutorials/00-application-connectivity/ac-04-register-secured-api.md

```diff
@@ -118,6 +118,44 @@ To create such a Secret, run this command:
 ```bash
 kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal clientSecret={CLIENT_SECRET} -n kyma-integration
 ```
+## Register an OAuth-secured API with token obtained from client certificate-secured endpoint
```

Comment:
```suggestion
## Register an OAuth-secured API with a token obtained from a client certificate-secured endpoint
```

Wouldn't this be more consistent with the description at the beginning of the document, though? 
```
## Register an OAuth 2.0 mTLS-secured API 
```

Reply:
Totally no problem with that. Please feel free to change it